frontend/public/locales/en/translation.json

```diff
@@ -3385,6 +3386,7 @@
   "You don't have any policy sets": "You don't have any policy sets",
   "You don't have any subscriptions": "You don't have any subscriptions",
   "You must fill out all required fields before you can proceed.": "You must fill out all required fields before you can proceed.",
+  "You must fix editor syntax errors.": "You must fix editor syntax errors.",
```

Comment:
@jeswanke Why do we have two versions of this string. This one, and the polite one?

@oafischer Can you weigh in on whether we should use "please" in these cases?

@jeswanke Some screenshots would be helpful to show Oliver (and any PR reviewer) where these strings appear. Thanks.

Reply:
We should generally avoid using "please". I'll take a look at the screenshots as well once they are available.

Reply:
@oafischer here's the two spots-- so one wizard is polite and the other isn't:
![CleanShot 2025-05-15 at 10 58 44@2x](https://github.com/user-attachments/assets/fa2e7cb9-1d34-43cb-a855-e3f907b83260)
![CleanShot 2025-05-15 at 11 00 30@2x](https://github.com/user-attachments/assets/ba4de5dc-02aa-4be1-8fda-6a6babb6203d)